File: fm-domain/src/main/java/io/zeitwert/fm/building/adapter/api/rest/dto/NoteTransferDto.java
package io.zeitwert.fm.building.adapter.api.rest.dto;

import lombok.Builder;
import lombok.Data;

import java.time.OffsetDateTime;

@Data()
@Builder
public class NoteTransferDto {

	private String subject;
	private String content;
	private Boolean isPrivate;

	private String createdByUser;
	private OffsetDateTime createdAt;

	private String modifiedByUser;
	private OffsetDateTime modifiedAt;

}

File: fm-domain/src/main/java/io/zeitwert/fm/building/api/rest/dto/NoteTransferDto.kt
package io.zeitwert.fm.building.api.rest.dto

import java.time.OffsetDateTime

data class NoteTransferDto(
	var subject: String? = null,
	var content: String? = null,
	var isPrivate: Boolean? = null,
	var createdByUser: String? = null,
	var createdAt: OffsetDateTime? = null,
	var modifiedByUser: String? = null,
	var modifiedAt: OffsetDateTime? = null,
)
